dbt_invoke/internal/_utils.py

```diff
@@ -249,12 +257,12 @@ def dbt_run_operation(
         macro_kwargs = macro_kwargs.replace("'", """'"'"'""")
         macro_kwargs = f"'{macro_kwargs}'"
     command = (
-        f"dbt run-operation {dbt_cli_kwargs}"
+        f"dbt {dbt_global_cli_kwargs} run-operation {dbt_cli_kwargs}"
         f" {macro_name} --args {macro_kwargs}"
     )
     logger.debug(f'Running command: {command}')
     result = ctx.run(command, hide=hide)
-    result_lines = result.stdout.splitlines()[1:]
+    result_lines = [json.loads(data) for data in result.stdout.splitlines()]
```

Comment:
I think there are at least two choices here:
1. If this line is going to assume json, then we can get rid of the `log_format` argument and hardcode `'json'` in line 243.
2. Or we can keep the `log_format` argument and fall back to `result_lines = result.stdout.splitlines()` when `log_format != 'json'`

Reply:
I prefer the hardcode version. I can't think of a reason where not using json log_format in the context of this function.

Reply:
Works for me